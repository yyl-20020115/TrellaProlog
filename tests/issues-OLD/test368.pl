main :-
	writeq((1*2)/(3*4)), nl,
	writeq(-(-a)), nl,
	writeq(-(1)), nl,
	writeq(-(-1)), nl,
	writeq(-(-(-a))), nl,
	writeq(-(-(1))), nl,
	writeq(-(-1- -1)), nl,
	writeq(-(-1- +1)), nl,
	writeq(-(-1- 1)), nl,
	writeq(-(-(-a))), nl,
	writeq(+(+1)), nl,
	writeq(+(+)), nl,
	writeq(-(-)), nl,
	writeq(-(-1- +a)), nl,
	writeq(-(-1- -a)), nl,
	true.

:- initialization(main).
