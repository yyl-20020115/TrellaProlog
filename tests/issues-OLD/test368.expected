1*2/(3*4)
- -a
- (1)
- -1
- - -a
- - (1)
- (-1- -1)
- (-1- +1)
- (-1-1)
- - -a
+ +1
+(+)
- (-)
- (-1- +a)
- (-1- -a)
